app/components/chart-footnotes.tsx

```diff
@@ -174,6 +174,11 @@ export const ChartFootnotes = ({
                 dangerouslySetInnerHTML={{ __html: dataCubeByIri.publisher }}
               ></Box>
             )}
+            {configKey && shareUrl && visualizeLinkText && (
+              <>
+                / <LinkButton href={shareUrl}> {visualizeLinkText}</LinkButton>
+              </>
+            )}
```

Comment:
Wondering if there will not be a space missing before the slash ?

Reply:
Good catch, I missed that!